project/build.scala

```diff
@@ -396,6 +405,8 @@ object build {
     }
   }
 
+  lazy val licenseFile = settingKey[File]("The license file to include in packaged artifacts")
```

Comment:
no need for `lazy`, I think it's cargo culting at this point :smiley:

Reply:
It's 100% cargo culting I guess. Sorry :P

Reply:
it's not worth pushing an update to this PR, the CI uses a small diesel power plant ... think of the carbon.

Reply:
and thus are these myths perpetuated.